zero-waste-service/routes/bin.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express')
+const router = express.Router();
+const models = require('../models/index')
+
+router.post('/', function (req, res) {
+    const userId = req.body.userId;
+    if(userId == null) {
+        throw new Error("Input format is incorrect")
+    }
+
+    const bin = models.bin;
+    bin.find()
+    .where("userId")
+    .equals(userId)
+    .select("name userId status")
+    .exec((err, result) => {
+        if (err) return "Error with fetching data";
+
+        return res.status(200).send(result);
+    })
+})
+
+router.post('/save', function (req, res) {
+    const name = req.body.name;
+    const pid = req.body.pid;
+    const userId = req.body.userId;
+    const status = req.body?.status ?? 'Fillable';
```

Comment:
Is this the right syntax?

Reply:
Yes it is optional chaining. It will by default take value 'Fillbale' when status is not supplied in the body

Reply:
Got it.